sematic/ui/packages/common/src/typeViz/views/hugging_face_stored_model.tsx

```diff
@@ -0,0 +1,27 @@
+import { ValueComponentProps, ViewComponentProps} from "src/typeViz/common";
+
+export function HuggingFaceStoredModelShortView(props: ValueComponentProps) {
+    return HuggingFaceStoredModelView(props, true);
```

Comment:
I recommend using JSX syntax here instead of using a function. That way, we treat `HuggingFaceStoredModelView` as a React component and can use hooks and provide better extensibility for the future.

To achieve that, simply do:

```
   <HuggingFaceStoredModelView {...props} short={true} />;
```

and change the call signature of `HuggingFaceStoredModelView ` to 
```
HuggingFaceStoredModelView(props: ViewComponentProps & { short: boolean } ) 
```

or to make it cleaner, define the props type explicitly.
```
interface HuggingFaceStoredModelViewProps extends ViewComponentProps {
    short: boolean
}
HuggingFaceStoredModelView(props: HuggingFaceStoredModelViewProps ) 

```

Reply:
done